### PR TITLE
Update sidebars.js

### PR DESCRIPTION
**Pull request by pstack2021**
@jeansebastienh can you have a look please, for now I just want to get the labels displaying in the TOC, I have created a new branch off the 'switch-to-docusaurus' branch, as there may be formatting issues in the javascript that you might need to fix - many thanks

### CODE REVIEW

File: sidebars.js [hunk 1]
```diff
@@ -18,6 +18,59 @@ const sidebars = {
 
   // But you can create a sidebar manually
   docsSidebar: [
+    {
+      type: 'category',
```
Comment (dumbbell) [inline sidebars.js:22]: You should use the type `link` when you want to specify an href.

File: sidebars.js [hunk 2]
```diff
@@ -18,6 +18,59 @@ const sidebars = {
...
+      label: 'Support Available for RabbitMQ',
+      href: 'https://www.rabbitmq.com/documentation.html',
+    }
+    {
+      type: 'category',
+      label: 'What is New in RabbitMQ Releases',
+      items: [
+        'whats-new',
```
Comment (dumbbell) [inline sidebars.js:40]: The `whats-next.md` page will have to exist for this to work.

File: sidebars.js [hunk 3]
```diff
@@ -18,6 +18,59 @@ const sidebars = {
...
+        {
+          type: 'doc',
+          label: 'What is New in RabbitMQ Releases',
+          items: [
```
Comment (dumbbell) [inline sidebars.js:44]: A `doc` entry can't have items. The type should be `category` here.

**Review by dumbbell — CHANGES_REQUESTED** [review]

**Comment by michaelklishin** [comment]
What's this "What's new" file? Is it a version of the change log? is it a blog post? Why does it exist in a PR that's meant to change something else?

**Pull request merged**